### PR TITLE
Add option -p  to set  the # of cpus on stringtie

### PR DESCRIPTION
**Pull request by JoseEspinosa**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `<SOFTWARE>.version.txt` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/stringtie/stringtie/main.nf [hunk 1]
```diff
@@ -48,6 +48,7 @@ process STRINGTIE {
         -A ${prefix}.gene.abundance.txt \\
         -C ${prefix}.coverage.gtf \\
         -b ${prefix}.ballgown \\
+        -p $task.cpus \\
         $options.args
 
     stringtie --version > ${software}.version.txt
```
Comment (drpatelh) [inline modules/stringtie/stringtie/main.nf:54]: Fancy bumping the version too whilst you are here?
https://anaconda.org/bioconda/stringtie

Comment (drpatelh) [reply modules/stringtie/stringtie/main.nf:54]: Need to change the version command to use `echo` too.

Comment (JoseEspinosa) [reply modules/stringtie/stringtie/main.nf:54]: Sure! Did not realised :sweat:

File: modules/stringtie/merge/main.nf [hunk 2]
```diff
@@ -24,12 +24,15 @@ process STRINGTIE_MERGE {
...
+    path  "*.version.txt"      , emit: version
 
     script:
     """
     stringtie \\
         --merge $stringtie_gtf \\
         -G $annotation_gtf \\
         -o stringtie.merged.gtf
+
+    stringtie --version > ${software}.version.txt
```
Comment (drpatelh) [inline modules/stringtie/merge/main.nf:36]:
```suggestion
    echo \$(stringtie --version 2>&1) > ${software}.version.txt
```

File: modules/stringtie/stringtie/main.nf (hunk 1 above)
Comment (drpatelh) [inline modules/stringtie/stringtie/main.nf:54]:
```suggestion
    echo \$(stringtie --version 2>&1) > ${software}.version.txt
```

**Review by drpatelh — APPROVED** [review]
LGTM!

**Comment by JoseEspinosa** [comment]
Merged, it took more changes than what I expected :stuck_out_tongue_closed_eyes:

**Pull request merged**